docs/src/styles.css

```diff
@@ -1,18 +1,119 @@
 /* Dark mode colors. */
 :root {
-  --sl-color-accent-low: #074947;
+  /* --sl-color-accent-low: #074947;
   --sl-color-accent: #3fc6bf;
-  --sl-color-accent-high: #80d9d6;
+  --sl-color-accent-high: #80d9d6; */
+  --sl-font: 'Inter Variable', sans-serif;
+  font-feature-settings: 'ss01', 'ss02', 'case', 'salt';
+}
+
+html {
+  font-feature-settings: 'ss01', 'ss02', 'case', 'salt';
 }
 
 /* Light mode colors. */
 :root[data-theme='light'] {
-  --sl-color-accent-low: #55b3af;
-  --sl-color-accent: #047771;
-  --sl-color-accent-high: #07524f;
+  /* --accent-hue: 180;
```

Comment:
I think this file needs a bit of cleanup? I'm assuming you puts the CSS vars in comments here to make it easy to lookup without having to tab in/out of browser vs editor, but correct me if I'm wrong in that assumption ;)

Does it get removed by Astro/Vite when building the docs site, e.g. CSS minify step?